httpclient/src/test/kotlin/tbdex/sdk/httpclient/TbdexHttpClientTest.kt

```diff
@@ -91,7 +91,7 @@ class TbdexHttpClientTest {
 
     server.enqueue(MockResponse().setResponseCode(HttpURLConnection.HTTP_ACCEPTED))
 
-    val rfq = TestData.getRfq(pfiDid.uri, TypeId.generate("offering"))
+    val rfq = TestData.getRfq(pfiDid.uri, TypeId.generate("offering").toString())
```

Comment:
Nit: I think we can simplify here and other spots in this file because `getRfq()` generates a default value for the offering ID.
```suggestion
    val rfq = TestData.getRfq(pfiDid.uri)
```

Reply:
oh nice catch